components/ArtPiecesPreview/index.js

```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
 
-export default function ArtPiecePreview({ title, image, artist, slug }) {
+export default function ArtPiecesPreview({ title, image, artist, slug }) {
   return (
     <>
```

Comment:
The `<strong>` tag carries a specific semantic meaning, for quick styling purposes maybe `<b>` could be a better choice. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

Reply:
thanks for the hint!